.github/workflows/e2eTest.yml

```diff
@@ -32,3 +32,8 @@ jobs:
         with:
           skip-install: true
           args: "--color --hostname localhost \"electron:.\" \"./**/*.e2e.js\""
+      - name: Archive test screenshots
+        uses: actions/upload-artifact@v2
+        with:
+          name: test-screenshots
```

Comment:
Not yet. That's what I'm trying to do, testcafe has some problem with taking a screenshot on CI right now.
If I succeed it will be visible in action's run summary page, e.g https://github.com/ZbayApp/ZbayLite/actions/runs/1504112224